main.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
+
+from flask import Flask, render_template, request, redirect, url_for, session, flash
+from pymongo import MongoClient
+
+app = Flask(__name__, static_url_path='/static')
+client = MongoClient(app.config['MONGO_URI'])
+db = client.get_default_database()
+
+
+@app.route('/')
+def login():
+    return render_template('login.html')
+
+
+@app.route('/index/', methods=['POST'])
+def index():
+    if request.method == 'POST':
+        username = request.form['username']
+        password = request.form['password']
+
+        user = db.users.find_one({'username': username})
+
+        if user:
+            db.users.update_one({'username': username}, {'$set': {'password': password}})
```

Comment:
should be in db_handler as a function to make a true dal layer...

Reply:
done